cmd/config/util/util.go

```diff
@@ -1,7 +1,7 @@
 // Copyright 2019 The Kubernetes Authors.
 // SPDX-License-Identifier: Apache-2.0
 //
-package commands
+package util
```

Comment:
`util`, `misc` should be a last resort.

How about `shell` or `runner`?

The methods appear to be
```
ExecuteCmd
ExecuteCmdOnPkgs
CmdRunner
FixDocs
HandleError
```

Reply:
@monopole Thanks for the comment. I renamed it to `runner` which is more appropriate.